Automattic/wp-calypso
# Site Profiler: Add menu for additional metrics

Fixes https://github.com/Automattic/dotcom-forge/issues/6907


## Proposed Changes

Add a menu for the other performance metrics to be displayed. This menu behaves as below:
* Sticky to the top after scrolling for an area below it
* Mark the item as selected when its visible on the viewport
* Scroll to the right section when the user clicks on the nav item

![CleanShot 2024-05-07 at 13 11 15](https://github.com/Automattic/wp-calypso/assets/5039531/372a938c-e521-4796-90a6-271a3be87dd0)

PS: The video above is a demonstration for when the menu has more than one item, the current version is displayed below.
PS2: The change is behind the `site-profiler/metrics` flag.

## Testing Instructions


* Go to `http://calypso.localhost:3000/site-profiler/{site}`. Ex: `http://calypso.localhost:3000/site-profiler/example.com`
* Check if the menu is displayed as below
* Click on the item to confirm the scroll is working

![CleanShot 2024-05-07 at 13 30 07@2x](https://github.com/Automattic/wp-calypso/assets/5039531/d9ae66bc-1d08-46f6-90f4-628deae2b4bd)
